packages/Utils/src/ternary.ts

```diff
@@ -0,0 +1,4 @@
+const ternary = (condition: any, exprIfTrue: any, exprIfFalse?: any): any =>
```

Comment:
This kind of code is important to be tested

Reply:
I change it for the classNames library